app/src/main/res/menu/feedlist.xml

```diff
@@ -70,6 +69,12 @@
         </menu>
     </item>
 
+    <item
+        android:id="@+id/swipe_settings"
+        android:title="@string/swipeactions_label"
+        android:menuCategory="container"
+        custom:showAsAction="never" />
```

Comment:
Please make it a settings screen instead. The overflow menu currently shows settings related to that single feed, not general settings.

Reply:
Could you help, not really familiar with the settings code.
Besides I think this might issue sense because it is related to that screen (separate settings for inbox, episodes, feeds) and it's easier to switch, I think I would be easier to have it in the toolbar than hide it it in the settings, I guess that would only spark a lot of support questions like "how to change SwipeActions"

Reply:
> I think I would be easier to have it in the toolbar than hide it it in the settings

With that argument, we would also have to add settings like "show covers in lists", "prefer streaming", "enqueue downloads", etc to the overflow menus. Reducing the number of things that are on screen during normal usage is a pretty big priority for me. The fewer buttons/icons/etc, the easier it is for humans to "parse" the screen. So this should really be a setting instead. I can have a look at the code later. Basically, a new preference would get an onPreferenceClickListener that shows the dialog.

Reply:
ok would be glad if you could add the setting at the right place, there also needs to be a a TAG for the screen that the SwipeActions belong to, so maybe there needs to be a dialog like "which swipeaction do you want to change" and only afterwards the actual Settingsdialog is displayed.

Reply:
Done :)